### DISCUSSION TITLE
Kinderwagen / Stroller

### DISCUSSION THREAD

**Discussion by puerki** [streetcomplete/StreetComplete#5989] [Ideas]
Can you add a Task for stroller accessibility or some tasks for Children or Baby things? Thanks very much

**Comment by mnalis** [comment]
There is already a [quest asked on steps](https://wiki.openstreetmap.org/wiki/StreetComplete/Quests#steps_ramp)  whether they have Stroller (or other) ramp. And there is a quest about [playground access rights](https://wiki.openstreetmap.org/wiki/StreetComplete/Quests#playground_access).

Also, if you enable `Things Overlay` (second icon from top right to the left) you can micromap playground equipment (seesaw, slide, merry-go-round, swing, springy, climbing frame, etc. -- search for _"play"_ to find more)... And in `Places overlay` you can add missing Indoor Play centers, Childcare, Kindergarten, etc. -- search for _"child"_ to find more)..

If you have other ideas what would be useful, please do let us know!
